controller_manager/test/test_controller_failed_init/test_controller_failed_init.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
```

Comment:
Please update the copyright to your company name or "ros2_control development team" if you like. And also the year.

Reply:
```suggestion
// Copyright 2020 ros2_control development team
```